FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Device/FRDeviceCollectorTests.swift

```diff
@@ -81,7 +81,7 @@ class FRDeviceCollectorTests: FRAuthBaseTest {
         XCTAssertNotNil((hardware["cpu"] as? Int))
 //        XCTAssertNotNil((hardware["activeCPU"] as? Int))
         XCTAssertNotNil((hardware["memory"] as? Int))
-        XCTAssertNotNil((hardware["storage"] as? Int))
+//        XCTAssertNotNil((hardware["storage"] as? Int))
```

Comment:
Update copyright

Reply:
Also, would nice to have a short comment why it has been commented out

Reply:
Fixed